src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -8,360 +8,710 @@ authors:
-or "Roboto Italic." In other words, a typeface is what you _see_, and the font is
-what you _use_.
+The terms font and typeface are often used interchangeably by developers.
+However there is a difference: A typeface is the underlying visual design that
+can exist in many different typesetting technologies, and a font is one of these
+implementations, in a digital file format. In other words, a typeface is what
+you _see_, and the font is what you _use_.
+
+Another concept that is often overlooked is the distinction between a style, and
+a family. A style is a single and specific typeface, such as Bold Italic, and a
+family is the complete set of styles.
+
+Before variable fonts, each style was implemented as a separate font file. With
+variable fonts, all styles can be contained in a single file.
 
 <figure class="w-figure">
-  <img src="robot-specimen.png" alt="A specimen of Roboto">
-  <figcaption class="w-figcaption">
-    Examples of the Roboto typeface that was designed and developed
-    Christian Robertson.
+  <img src="roboto-family-specimen.png" style="max-width: 100%;" alt="A specimen composition and list of different styles of the Roboto family">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Above: Left, a specimen of the Roboto typeface family. Right, named styles within the family.
   </figcaption>
 </figure>
 
+## Challenges for the Designer and Developer
+
+<!--
+RN: Are we still doing print industry references in 2020? How many web developers would relate to that in an article called "Introduction to variable fonts on the web"?
+
+Cut to the chase?
+
+Unlike books or magazines, which can have as many fonts on a page as they like, a website will have to download every single font it uses. This has been a sticking point for richer typographic experiences, as each font comes at a cost.
+
+With traditional web fonts... etc.
+-->
+
+When a designer creates a print project they face some constraints, such as the
+physical size of the page layout, the number of colors they can use is
+determined by the kind of printing press that will be used, and so on. But they
+can use as many typeface styles as they like. This means that the typography of
+print media is often rich and sophisticated, so that the reading experience is
+truly delightful. Think of the last time you enjoyed browsing an excellent
+magazine.
+
+Web designers and developers have different constraints than print designers,
+and an important one is the associated bandwidth costs of our designs. This has
+been a sticking point for richer typographic experiences, as they come at a
+cost. With traditional web fonts, each style used in our designs requires users
+to download a separate font file, which increases latency and page rendering
+time. Only including the Regular and Bold styles, plus their italic
+counterparts, can amount to 500k or more of font data. This is even before we
```

Comment:
```suggestion
counterparts, can amount to 500 KB or more of font data. This is even before we
```
I can't find a reference in style guide but I'm pretty sure we use KB for kilobytes and Kb for kilobits